### PR TITLE
Project: Weather App

### PR DESCRIPTION
**Pull request by somoye123**
I built a Weather App using the Weather API using the async and await functions. I can change the location by filling the search field and toggle between Celsius and Fahrenheit by clicking on their respective buttons, I built the classes and dom actions for the weather.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @somoye123 
### You have done well so far :clap: However, you have a few issues to fix:
- [ ] Update your scripts dev, see below
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
- [ ] In the setup section in the readme file, be detailed i.e `git clone git@github.com:somoye123/Weather-app.git`
### Make the changes and submit for another code review. Good luck.
- Happy coding!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- Clone the repository and there you go! ;-)
+
+### Prerequisites
+
+- Get a browser like Chrome and Firefox in their most recent versions
+
+### Setup
+
+- git clone https://github.com/somoye123/Weather-app.git
+- Weather-app
```
Comment (mosesogwo) [inline README.md]: What happens in this step? I guess you should have a `cd` there.

**Review by mosesogwo — APPROVED** [review]
**Well done.:+1: You can merge after fixing the comment on your README**

_**Good luck:stars:**_

### Status: Approved :heavy_check_mark:

File: README.md (hunk 1 above)
Comment (somoye123) [reply README.md]: thanks
added the cd

**Pull request merged**